localization/beluga_demo_ndt_3d_localization/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023-2024 Ekumen, Inc.
```

Comment:
@Fernando-Sanz nit: dates need an update, I think. Here and elsewhere in this patch.